adapter-cqld4/src/main/java/io/nosqlbench/cqlgen/model/CqlModel.java

```diff
@@ -75,18 +75,18 @@ public void setKeyspaceAttributes(CGSchemaStats schemaStats) {
 
             CqlKeyspaceDef ksdef = getKeyspace(statsKeyspacename);
             if (ksdef !=null) {
-                logger.debug(() -> "setting         keyspace stats for '" + statsKeyspacename + "'");
+                logger.debug(() -> STR."setting         keyspace stats for '\{statsKeyspacename}'");
```

Comment:
Can we remove extra tabs, I guess it's by mistake in original codebase.

Reply:
done